app/controllers/gcs_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'google/cloud/storage'
+
+class GcsController < ApplicationController
+
+  # POST /gcs
+  def create
+
+    bucket_name = 'scl-lny-receipts'
+    file_name = gcs_params[:file_name]
```

Comment:
since this is a shared bucket and it looks like the client is updating the filename? if theres a convention i recommend storing possibly a DATESTAMP_VENDOR_NAME or something of the like to avoid conflicts. 

Also you can use possibly folders/prefixes but this should be ok since if we expect < 5000 receipts it should be managable

Reply:
My idea was to have the front-end construct the filename as it has the relevant information (user email address, vendor name, etc.). I think with our scale all in the same bucket without files is probably fine for now, but we can update later if need be. In the front-end MR I did `[email before @ sign]-[full ISO timestamp].[ext]`, but maybe `[full-escaped-email]-[vendor name]-[full ISO timstamp].ext` will actually be better. What do you think?